src/ext/oblivc/obliv_bits.c

```diff
@@ -56,7 +56,6 @@ static struct stdioTransport stdioTransport
 void protocolUseStdio(ProtocolDesc* pd)
   { pd->trans = &stdioTransport.cb; }
 
-//#define PROFILE_NETWORK
```

Comment:
It's not important that it _always_ be enabled, but rebuilding every time you need network profiling is fairly inconvenient (and it seems to be not an uncommon need - I've been repeatedly asked for network figures for various things), and adding a runtime conditional for each gate seems like it wouldn't be significantly more efficient than the single extra addition that permanent enabling incurs. In other words, if it doesn't incur significant overhead (seems like it should be about the same, overhead-wise, as counting gates), is there a reason to disable it? Building two versions of libobliv is definitely better than being required to rebuild libobliv, but it still implies rebuilding of the application binaries, dealing with flags and linking, &c, and I'm not sure what the benefit would be of linking against the one _without_ network profiling.

Please let me know if I've failed to understand something. My assertion that it causes negligible overhead is based on the metric of running programs with it enabled and not noticing any, rather than any rigorous measurement.